content/articles/how-to-create-a-user-login-page-using-javagui/index.md

```diff
@@ -0,0 +1,136 @@
+### How to create a user login page using Java GUI
+
+### Introduction
+
+Java is an Object-Oriented Programming Language. G.U.I stands for Graphical User Interface. GUIs are made up of objects and classes that can interact with each other. GUIs are said to be `Event Driven` programs. this means that GUIs depends on series of object events that can occur at any given time.
+
+### Goals
+
+In this tutorial, we will learn how to design a simple user login page. At the end of this tutorial, the reader will be able to design the user login page below.
+
+![Login page](/how-to-create-a-user-login-page-using-java-gui/loginPage.png)
+
+### Prerequisites
+
+1. Basic Understanding of Java and Java GUIs.
+2. [JDK version 16 or higher](www.oracle.org).
+3. Java compiler of your choice ( Recommended: [Eclipse](www.eclipse.org) )
+
+### Setup procedure
+
+Open eclipse, click on File, select New, then select Class, name your class by Java naming convention.
+If done properly you will have a window like this with the given class name, in this tutorial we name our class Java_GUI.
+
+![java gui](/how-to-create-a-user-login-page-using-java-gui/structure.png)
+**NOTE:** We will import `javax.swing.*;`,`import java.awt.event.*;` this will cover the rest of our java swing extension classes and event package.
+
+#### Variables
+Now let's create our private static variables class and also set it to implements an action whenever the button is click.
```

Comment:
This statement gives an overview. More details could be added.

Like, why private variables? What is static? What action listener are you building? Why?